openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/salesforceConnection.json

```diff
@@ -26,12 +26,6 @@
       "$ref": "#/definitions/salesforceType",
       "default": "Salesforce"
     },
-    "scheme": {
```

Comment:
salesforce is not an SQA source hence scheme is not required

Reply:
Deleting the schema might have caused issue #11468, could you please take a look?